site/specs-source/voice.json

```diff
@@ -2781,34 +2804,42 @@
           },
           "parentCallId": {
             "nullable": true,
-            "type": "string"
+            "type": "string",
+            "description": "The A-leg call id, set only if this call is the B-leg of a [`<Transfer>`](/docs/voice/bxml/transfer)."
           },
           "applicationId": {
-            "type": "string"
+            "type": "string",
+            "description": "The application id associated  with the call."
           },
           "accountId": {
-            "type": "string"
+            "type": "string",
+            "description": "The account id associated with the call."
           },
           "to": {
-            "type": "string"
+            "type": "string",
+            "description": "The phone number that received the call, in E.164 format (e.g. +15555555555), or if the call was to a SIP URI, the SIP URI"
           },
           "from": {
-            "type": "string"
+            "type": "string",
+            "description": "The phone number that made the call, in E.164 format (e.g. +15555555555)."
           },
           "direction": {
-            "type": "string"
+            "type": "string",
+            "description": "The direction of the call. Either `inbound` or `outbound`."
           },
           "state": {
             "description": "The current state of the call. Current possible values are 'queued', 'initiated', 'answered' and 'disconnected'. Additional states may be added in the future, so your application must be tolerant of unknown values.",
             "type": "string"
           },
           "identity": {
             "nullable": true,
-            "type": "string"
+            "type": "string",
+            "description": "The value of the `Identity` header from the inbound invite request. Only present for inbound calls and if the account is configured to forward this header."
           },
           "stirShaken": {
             "nullable": true,
             "type": "object",
+            "description": "For inbound calls, the Bandwidth STIR/SHAKEN implementation will verify the information provided in the inbound invite request `Identity` header. The verification status is stored in the call state `stirShaken` property as follows.\n\n| Property          | Description |\n|:------------------|:------------|\n| verstat | (optional) The verification status indicating whether the verification was successful or not. Possible values are `TN-Verification-Passed` or `TN-Verification-Failed`. |\n| attestationIndicator | (optional) The attestation level verified by Bandwidth. Possible values are `A` (full), `B` (partial) or `C` (gateway). |\n| originatingId | (optional) A unique origination identifier. |\n\nMore information: [Understanding STIR/SHAKEN](https://www.bandwidth.com/regulations/stir-shaken)\n",
```

Comment:
for some reason, `originatingId` is missing the bottom line on the page.

Reply:
Not sure what you mean, looks correct to me
![image](https://user-images.githubusercontent.com/12836996/148243077-57fa255f-90de-4354-be8f-a2ce672166d9.png)